### PR TITLE
Feat/backend lockdown improvements

### PR DESCRIPTION
**Pull request by ilyasfoo**
Prerequisite PR: https://github.com/Code-for-All/lockdown/pull/128

Summary:
- Changed to use snapshot format
- Until snapshots are ready, all territories will use [ASM_Fetch sheet](https://docs.google.com/spreadsheets/d/1mVyQxxLxAF3E1dw870WHXTOLgYzmumojvzIekpgvLV0/edit#gid=418075319)
- Changed to transform values using shared types (default is null)
- Changed latest entry definition: current date at UTC+0

### CODE REVIEW

File: backend/src/loaders/lockdown/lockdown.js [hunk 1]
```diff
@@ -176,39 +162,51 @@ function getEntry(sheet, entryIndex) {
...
     logger.log(`[Lockdown:WorkSheet] ${batch.map(t => t['iso3']).join(' ')}`);
     let gridData = await doc.batchGetGridRanges(gridRanges);
     
     for (let i = 0; i < batch.length; i++) {
       // TODO: Uncomment the following when country tab sheets are ready with ISO3 naming
-      // let workSheet = await getWorksheetByTitle(batch[i]['iso3']);
-      let workSheet = await getWorksheetByTitle('DEMO');
-      let gridSheet = new SimpleGridSheet(rangeToCache, gridData[i], workSheet);
+      // let workSheet = await getWorksheetByTitle(`${batch[i]['iso3']}_Fetch`);
+      let workSheet = await getWorksheetByTitle('ASM_Fetch');
```
Comment (LarsDenBakker) [inline backend/src/loaders/lockdown/lockdown.js:185]: Using await in a loop makes the async processing sequential, while we could parallelize this work. is this intentional?

**Comment by LarsDenBakker** [comment]
Defaulting to null makes it appear in the generated JSON, while defaulting to undefined will not make it appear saving a bit on the final size. @thepassle I think you specifically requested null before?

**Comment by LarsDenBakker** [comment]
great to see we got this updated to quickly 👍 will really help move things along

File: backend/src/loaders/lockdown/lockdown.js (hunk 1 above)
Comment (ilyasfoo) [reply backend/src/loaders/lockdown/lockdown.js:185]: `getWorksheetByTitle` is a lazy-loaded function. The intention of using async was just for code readability, parallelizing this wouldn't make much of an impact. Most intensive operations are between these [lines](https://github.com/Code-for-All/lockdown/pull/129/files#diff-4c1a2a4721c4db3c9e706007353b0ce2R192-R198)

**Comment by thepassle** [comment]
> Defaulting to null makes it appear in the generated JSON, while defaulting to undefined will not make it appear saving a bit on the final size. @thepassle I think you specifically requested null before?

Is this referring to the label of a measure, or the value? I think returning null is more 'correct', but I do agree about the size. How would we handle it in the frontend though? We'd need a list of the labels (that is up to date with the labels provided by the json/google sheet), and if a label is not there, we'd need to figure out which label is not there somehow, and still render it as being 'no data'

If this is referring to the value of a measure, I think we can do omit it, and then on the frontend do something like `measure?.value ?? 'empty'`

**Comment by LarsDenBakker** [comment]
@passle can you give an example of a list of labels?

**Comment by thepassle** [comment]
> @passle can you give an example of a list of labels?

```js
{
   "measures":[
      {
         "label":"max_gathering",
         "value":3
      },
      {
         "label":"lockdown_status",
         "value":"3"
      }
   ]
}```

**Pull request merged**